SECURITY.md

```diff
@@ -0,0 +1,87 @@
+<!-- Inherited from https://github.com/celo-org/celo-monorepo/blob/master/SECURITY.md & https://github.com/celo-org/social-connect/blob/main/SECURITY.md -->
+
+# Security
+
+## Security Announcements
+
+> [!IMPORTANT]
+> Public announcements of new releases with security fixes and of disclosure of any vulnerabilities will be made in the Celo Forum's [Security Announcements](https://forum.celo.org/c/security-announcements/) channel.
+
+# Reporting a Vulnerability
+
+ We’re extremely grateful for security researchers and users that report vulnerabilities to the Celo community. All reports are thoroughly investigated.
+
+> [!CAUTION]
+> **Please do not create a public ticket** mentioning any vulnerability.
+
+The Celo community asks that all suspected vulnerabilities be privately and responsibly disclosed.
+
+## Creating a Report
+
+1. Submit your vulnerability to [Celo on Intigriti](https://app.intigriti.com/programs/clabs/clabs/detail). This is currently a public program
```

Comment:
This is the part that caught my eye. Seems like this might not be the only provider (we use [hexens](https://hexens.io/) as of late), or might not be a provider at all anymore.

Reply:
This is fixed here https://github.com/celo-org/.github/pull/1/commits/44162e19f4e0d2a8c1aca59d2312d8ff82703852